about-us.html

```diff
@@ -0,0 +1,137 @@
+              </svg>
+            </li>
+            <li class="white-text">English</li>
+            <li class="white-text">My Page</li>
+            <li class="white-text">logout</li>
+          </ul>
+        </div>
+      </nav>
+    </section>
+    <section id="main-navigation" class="nav-section">
+      <div class="container">
+        <nav class="main-navigation">
+          <div class="navigation-logo">
+            <a href="./index.html">
+              <svg height="20" width="20" xmlns="http://www.w3.org/2000/svg" class="h-5 w-5" viewBox="0 0 20 20" fill="currentColor">
+                <path fill-rule="evenodd" d="M4 2a2 2 0 00-2 2v11a3 3 0 106 0V4a2 2 0 00-2-2H4zm1 14a1 1 0 100-2 1 1 0 000 2zm5-1.757l4.9-4.9a2 2 0 000-2.828L13.485 5.1a2 2 0 00-2.828 0L10 5.757v8.486zM16 18H9.071l6-6H16a2 2 0 012 2v2a2 2 0 01-2 2z" clip-rule="evenodd" />
+              </svg>
+            </a>
+          </div>
+          <div class="main-menu">
+            <ul>
+              <li><a href="./index.html"> Home </a></li>
+              <li><a href="./about-us.html" class="last-child"> About Us </a></li>
+            </ul>
+          </div>
+          <div class="menu-icon">
+            <svg height="20" width="20" xmlns="http://www.w3.org/2000/svg" class="h-6 w-6" fill="none" viewBox="0 0 24 24" stroke="currentColor">
+              <path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M4 6h16M4 12h16M4 18h16" />
+            </svg>
+          </div>
+        </nav>
+      </div>
+      <div class="mobile-navigaton hide">
+        <div class="close-btn">
+          <svg height="20" width="20" xmlns="http://www.w3.org/2000/svg" class="h-6 w-6" fill="none" viewBox="0 0 24 24" stroke="currentColor">
+            <path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M6 18L18 6M6 6l12 12" />
+          </svg>
+        </div>
+        <div class="mobile-menu">
+          <ul>
+            <li> <a href="./index.html">Home</a></li>
+            <li> <a href="./about-us.html">About Us </a></li>
+          </ul>
+        </div>
+      </div>
+    </section>
+    <section id="heading">
+      <div class="head">
+        <div class="container">
+          <div class="head-content">
+            <h2 class="title-about">Tourism Events Global summit <span class="year"> 2022 </span></h2>
```

Comment:
- [ ] To keep inline with correct HTML semantics we should have one `h1` tag on every page. In the about page you don't have a `h1` tag. please implement one.